src/components/strategies/portfolio/PortfolioMobileCard.tsx

```diff
@@ -40,7 +42,12 @@ export const PortfolioMobileCard: FC<Props> = ({
 
   if (href) {
     return (
-      <Link to={href} className={wrapperClasses}>
+      <Link
+        to={href}
+        params={params ?? {}}
```

Comment:
Should we add a `TODO: fix it` here too ?

Reply:
jupp